src/Table/sections/helperFunctions.ts

```diff
@@ -0,0 +1,97 @@
+import { TableRow, sortDirectionTypes, TableHeader, Cell } from "../Table";
+
+/**
+ * sum the total of columns or cols in a row
+ * @param colsLength the length of the columns
+ * @param useSelection add a column for selection checkboxes
+ * @param useShowActionColumn add another column for action columns
+ * @param useGroupBy add another columns for groupby
+ */
+export function sumCols(colsLength: number, useSelection?: boolean, useShowActionColumn?: boolean, useGroupBy?: boolean): number {
+    let sum = colsLength;
+
+    if (useSelection) {
+        sum = sum + 1;
+    }
+
+    if (useGroupBy) {
+        sum = sum + 1;
+    }
+    if (useShowActionColumn) {
+        sum = sum + 1;
+    }
+
+    return sum;
+}
+
+/**
+ * sort array of tabke rows
+ * @param items table rows array
+ * @param columnName the target column name
+ * @param sortDirection the sort direction
+ * @return Array of tableRow
+ */
+export function sortArray(items: Array<TableRow> = [], columnName: string, sortDirection: sortDirectionTypes): Array<TableRow> {
+    const languages: Readonly<Array<string>> = window.navigator?.languages || ["sw", "en"];
```

Comment:
I don't think you should force `sw` and `en` on the component. You should find a way to get the language from the `navigator` or any other way.

Reply:
we are getting from navigator "sw" and "en" are just a fallback incase users browser doesn't support one.